verilog/tools/ls/README.md

```diff
@@ -236,3 +271,55 @@ code --install-extension verible.vsix
 ```
 
 [release]: https://github.com/chipsalliance/verible/releases
+
+## Configuring the Language Server for a project
+
+### Adding `verible.filelist` for project-wide symbol discovery
+
+`verible-verilog-ls` by default loads and analyses only currently edited files in the editor.
+To be able to utilize such features as going to definition or references project-wide, add a `verible.filelist` file to the project.
+`verible.filelist` is a file containing a list of design files to collect data necessary to build a symbol table and other helper structures to allow Language Server to provide definitions, references, information on hover and more.
+It is a simple file containing one file per line - it can be easily created using such tools as `find`, e.g.:
```

Comment:
Most people will already have a file list in some shape as this is what their other tooling is using.

So here I'd first describe that this is just a standard file list and if the tooling already has a file list, simply adding a symbolic link with the name `verible.filelist` pointing to the original file list should be sufficient.

Then, if that is not available, having the `find` method is a good fallback way of creating a file list.

Reply:
Ok, I suggested linking , and also provided a `--file_list_path` flag as a possibility to change the default name of the list file